data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,12 @@
+/* exported isEmpty */
+
+function isEmpty(stack) {
+  // if (stack.peek() === undefined) {
+  //   return true;
+  // }
+  // return false;
+
+  // return stack.peek() === undefined ? true : false
+  // why does line 11 pass?
+  return stack.peek() === undefined;
```

Comment:
This passes because the return of the peek method either is undefined (`true`) or it isn't (`false`)